src/features/formData/FormDataWrite.tsx

```diff
@@ -187,24 +189,24 @@ function useFormDataSaveMutation() {
             return;
           }
 
-          const { newDataModels, validationIssues } = await doPatchMultipleFormData(multiPatchUrl, {
+          const { newDataModels, validationIssues, instance } = await doPatchMultipleFormData(multiPatchUrl, {
             patches,
             // Ignore validations that require layout parsing in the backend which will slow down requests significantly
             ignoredValidators: IgnoredValidators,
           });
 
           const dataModelChanges: UpdatedDataModel[] = [];
-          for (const dataElementId of Object.keys(newDataModels)) {
+          for (const { id: dataElementId, data } of newDataModels) {
```

Comment:
The name should probably be dataElementId in the response? I think I’ll update the api